builtin/providers/aws/resource_aws_autoscaling_group.go

```diff
@@ -118,10 +119,73 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 					return hashcode.String(v.(string))
 				},
 			},
+			"tags": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": &schema.Schema{
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						"value": &schema.Schema{
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+
+						"propagate": &schema.Schema{
+							Type:     schema.TypeBool,
+							Required: true,
```

Comment:
The [ASG API reference][1] lists `propagate` as optional, is there a reason to mark this as required here? 


[1]: http://docs.aws.amazon.com/AutoScaling/latest/APIReference/API_CreateAutoScalingGroup.html